plugins/workspace-backpack/src/options.d.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/**
+ * @fileoverview Typedefs and utility methods for parsing options for the
+ * backpack plugin.
+ * @author kozbial@google.com (Monica Kozbial)
+ */
+export type BackpackOptions = {
+    allowEmptyBackpackOpen: boolean;
+    contextMenu: {
+        emptyBackpack: boolean;
+        removeFromBackpack: boolean;
+        copyToBackpack: boolean;
+        copyAllToBackpack: boolean;
+        pasteAllToBackpack: boolean;
+        disablePreconditionChecks: boolean;
+    };
+};
+export type BackpackContextMenuOptions = {
+    allowEmptyBackpackOpen?: boolean;
+    contextMenu?: BackpackContextMenuOptions;
```

Comment:
I don't think this type should be recursively referencing itself.

Reply:
Good catch. Not sure what happened to `BackpackContextMenuOptions `. Fixed and fixed.